server/src/main/java/com/linecorp/centraldogma/server/internal/admin/decorator/ProjectOwnersOnly.java

```diff
@@ -26,7 +28,7 @@
 public class ProjectOwnersOnly extends ProjectAccessController {
 
     @Override
-    protected boolean isAllowedRole(User user, ProjectRole projectRole) {
+    protected boolean isAllowedRole(User user, @Nullable ProjectRole projectRole) {
         return User.ADMIN == user || ProjectRole.OWNER == projectRole;
```

Comment:
`user == User.ADMIN` and `projectRole == ProjectRole.OWNER` seem to read better to me.